plugins/circleci/src/components/App.tsx

```diff
@@ -24,14 +24,13 @@ export const App = () => {
   return (
     <AppStateProvider>
       <>
-        <Switch>
-          <Route path="/circleci" exact component={BuildsPage} />
+        <Routes>
+          <Route path="/circleci" element={<BuildsPage />} />
```

Comment:
So now we can refactor this to just be `path="/"` right?

Reply:
Yes, if the encapsulating `Routes` has `basename` of `/circleci` then the `Route` can just use `/`